### DISCUSSION TITLE
🧙 Arkanum: A Code-Server container optimized for daily use.

### DISCUSSION THREAD

**Discussion by OCram85** [coder/code-server#5836] [Show and tell]
Hey folks,
I just wanted to thank you for all the work you put into code-server. - It's a wonderful, usable piece of software that everyone who likes code should appreciate. 🙏 

I've tried to optimize the code-server experience for daily use and decided to publish it. I hope it's useful for others and helps coding.

----

<p align="right">
  <img src="http://forthebadge.com/images/badges/built-with-love.svg">
  <img src="http://forthebadge.com/images/badges/for-you.svg">
</p>

<p align="center">
  <a href="https://arkanum.dev">
    <img
      src="https://github.com/OCram85/arkanum/blob/master/assets/social-logo.png?raw=true"
      alt="Container"
    >
  </a>
</p>

<h1 align="center">
  🧙 Arkanum ✨ 🌌 ☄️ 💥
</h1>

<p align="center">
... is a Code-Server container optimized for daily use.
</p>

**Comment by lg-nineoneninethreenine** [comment]
Interesting! Does this work on all operating systems? And how does it work?

**Reply by lg-nineoneninethreenine** [reply]
This is so cool! Can I just use the app without the container? Like build from source?

**Reply by OCram85** [reply]
hmm 🤔 ,

Arkanum docker image is based on the [linuxserver/docker-code-server](https://github.com/linuxserver/docker-code-server) project.
I started adding my default config files and customizing the files. You can find the changes in the [Dockerfile](https://github.com/OCram85/arkanum/blob/master/Dockerfile#L20) itself.

**Comment by jsjoeio** [comment]
This is very cool! Thank you for sharing! 🎉
